test/suites/test_msgpack_ext_types.py

```diff
@@ -0,0 +1,425 @@
+            'msgpack': (b'\x03\x1c'),
+            'tarantool': "decimal.new('0.001')",
+        },
+        {
+            'python': decimal.Decimal('99999999999999999999999999999999999999'),
+            'msgpack': (b'\x00\x09\x99\x99\x99\x99\x99\x99\x99\x99\x99' +
+                        b'\x99\x99\x99\x99\x99\x99\x99\x99\x99\x9c'),
+            'tarantool': "decimal.new('99999999999999999999999999999999999999')",
+        },
+        {
+            'python': decimal.Decimal('-99999999999999999999999999999999999999'),
+            'msgpack': (b'\x00\x09\x99\x99\x99\x99\x99\x99\x99\x99\x99' +
+                        b'\x99\x99\x99\x99\x99\x99\x99\x99\x99\x9d'),
+            'tarantool': "decimal.new('-99999999999999999999999999999999999999')",
+        },
+        {
+            'python': decimal.Decimal('-12.34'),
+            'msgpack': (b'\x02\x01\x23\x4d'),
+            'tarantool': "decimal.new('-12.34')",
+        },
+        {
+            'python': decimal.Decimal('12.34'),
+            'msgpack': (b'\x02\x01\x23\x4c'),
+            'tarantool': "decimal.new('12.34')",
+        },
+        {
+            'python': decimal.Decimal('1.4'),
+            'msgpack': (b'\x01\x01\x4c'),
+            'tarantool': "decimal.new('1.4')",
+        },
+        {
+            'python': decimal.Decimal('2.718281828459045'),
+            'msgpack': (b'\x0f\x02\x71\x82\x81\x82\x84\x59\x04\x5c'),
+            'tarantool': "decimal.new('2.718281828459045')",
+        },
+        {
+            'python': decimal.Decimal('-2.718281828459045'),
+            'msgpack': (b'\x0f\x02\x71\x82\x81\x82\x84\x59\x04\x5d'),
+            'tarantool': "decimal.new('-2.718281828459045')",
+        },
+        {
+            'python': decimal.Decimal('3.141592653589793'),
+            'msgpack': (b'\x0f\x03\x14\x15\x92\x65\x35\x89\x79\x3c'),
+            'tarantool': "decimal.new('3.141592653589793')",
+        },
+        {
+            'python': decimal.Decimal('-3.141592653589793'),
+            'msgpack': (b'\x0f\x03\x14\x15\x92\x65\x35\x89\x79\x3d'),
+            'tarantool': "decimal.new('-3.141592653589793')",
+        },
+        {
+            'python': decimal.Decimal('1234567891234567890.0987654321987654321'),
+            'msgpack': (b'\x13\x01\x23\x45\x67\x89\x12\x34\x56\x78\x90' +
+                        b'\x09\x87\x65\x43\x21\x98\x76\x54\x32\x1c'),
+            'tarantool': "decimal.new('1234567891234567890.0987654321987654321')",
+        },
+        {
+            'python': decimal.Decimal('-1234567891234567890.0987654321987654321'),
+            'msgpack': (b'\x13\x01\x23\x45\x67\x89\x12\x34\x56\x78\x90' +
+                        b'\x09\x87\x65\x43\x21\x98\x76\x54\x32\x1d'),
+            'tarantool': "decimal.new('-1234567891234567890.0987654321987654321')",
+        },
+    ]
+
+    def test_decimal_msgpack_decode(self):
+        for i in range(len(self.valid_decimal_cases)):
+            with self.subTest(msg=str(i)):
```

Comment:
It would be easier in the future to find a failed case by value, rather than count a number:
```suggestion
            with self.subTest(msg=str(self.valid_decimal_cases[i]['python'])):
```
The same in all tests.

Reply:
Used named test cases